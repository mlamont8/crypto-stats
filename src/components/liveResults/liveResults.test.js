import React from 'react';
import renderer from 'react-test-renderer';
import LiveResults from './liveResults';

describe('liveResults component', () => {
    it('renders correctly', () => {

        const renderedLive = renderer.create(
            <LiveResults
                dollar={7505.4}
                exchange="Binance"
                flag="1"
                price="0.0000379"
                to="BTC"
            />
        );
        expect(renderedLive.toJSON()).toMatchSnapshot();
    })
})
