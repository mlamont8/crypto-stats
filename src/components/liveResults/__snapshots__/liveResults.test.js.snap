// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`liveResults component renders correctly 1`] = `
<div
  className="current text-center infoBlock"
>
  <div
    className="blockTitle"
  >
    <h1>
      CURRENT
    </h1>
  </div>
  Binance
  <hr />
  <div
    className="row"
  >
    <div
      className="liveItem"
    >
      <span
        className="price"
      >
        0.0000379
      </span>
      <span
        className="toPrice"
      >
        BTC
      </span>
    </div>
  </div>
  <div
    className="row"
  >
    <span
      className="conversion"
      id="arrow-up"
    >
      ($
      0.28
      )
    </span>
  </div>
</div>
`;
